src/test/java/picard/sam/markduplicates/CheckDuplicateMarkingTest.java

```diff
@@ -0,0 +1,36 @@
+package picard.sam.markduplicates;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import picard.sam.CheckDuplicateMarking;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class CheckDuplicateMarkingTest {
+    private static final String TEST_FILES_DIR="testdata/picard/sam/CheckDuplicateMarking";
```

Comment:
Reformat new code to follow conventions
```suggestion
    private static final String TEST_FILES_DIR = "testdata/picard/sam/CheckDuplicateMarking";
```